src/main/java/edu/pw/eiti/pik/user/UserController.java

```diff
@@ -47,6 +54,25 @@ UserDto me() {
         return userMapper.toDto(user);
     }
 
+    @ApiOperation(value = "Get top users by name who have given role (up to number)")
+    @ApiResponses({
+    	@ApiResponse(code = 200, message = "Always")
+    })
+    @GetMapping(path = "/user/{authority}/{name}/{number}")
+    List<UserDto> getUsersByRoleAndName(
+    		@PathVariable String authority,
```

Comment:
This should be enum. Check out how to use enums in controllers in ParticipationController